templates/forespørsel-nous-eksamen.json

```diff
@@ -0,0 +1,36 @@
+{
+    "Title": "Svar på søknad om oppmelding til NUS-eksamen",
```

Comment:
Tittle på fila bør ikke inneholde "ø", og her er en "o" for mye ;) "templates/foresp**ø**rsel-n**o**us-eksamen.json"